akka-stream/src/main/scala/akka/stream/impl/fusing/LazySingleSource.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2022 Lightbend Inc. <https://www.lightbend.com>
+ */
```

Comment:
could you put Apache license on new files? 

```
/*
 *  ====================================================================
 *    Licensed to the Apache Software Foundation (ASF) under one or more
 *    contributor license agreements.  See the NOTICE file distributed with
 *    this work for additional information regarding copyright ownership.
 *    The ASF licenses this file to You under the Apache License, Version 2.0
 *    (the "License"); you may not use this file except in compliance with
 *    the License.  You may obtain a copy of the License at
 *
 *        http://www.apache.org/licenses/LICENSE-2.0
 *
 *    Unless required by applicable law or agreed to in writing, software
 *    distributed under the License is distributed on an "AS IS" BASIS,
 *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *    See the License for the specific language governing permissions and
 *    limitations under the License.
 * ====================================================================
 */
```

Reply:
I will check the header template then.

Reply:
Will resolved after https://github.com/apache/incubator-pekko/issues/38

Reply:
there is no action for #38 - the requirement is to keep Lightbend headers on existing files but to add Apache licenses on new files